doc/source/whatsnew/v1.1.0.rst

```diff
@@ -271,6 +271,21 @@ change, as ``fsspec`` will still bring in the same packages as before.
 
 .. _fsspec docs: https://filesystem-spec.readthedocs.io/en/latest/
 
+.. _whatsnew_110.binary_handle_to_csv:
+
+Support for binary file handles in ``to_csv``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:meth:`to_csv` supports file handles in binary mode if ``mode`` contains a ``b``.
+
+For example:
+
+.. ipython:: python
+
+   data = pd.DataFrame([0, 1, 2])
+   with open("output.csv", mode="w+b") as handle:
+       data.to_csv(handle, mode="w+b")
```

Comment:
add a suppress directive to remove the file

Reply:
I changed it to use a byte buffer instead